server/api/src/test/scala/cool/graph/api/server/RequestHandlerSpec.scala

```diff
@@ -18,13 +19,11 @@ import scala.concurrent.Future
 
 class RequestHandlerSpec extends FlatSpec with Matchers with ApiBaseSpec with AwaitUtils with GraphQLResponseAssertions {
   import system.dispatcher
-  import testDependencies.bugSnagger
+  import testDependencies.reporter
 
   "a request without token" should "result in an InvalidToken error" in {
-    val (_, result) = handler(projectWithSecret).handleRawRequestForPublicApi(projectWithSecret.id, request("header")).await
```

Comment:
Note: Error handling moved up one layer, meaning that here it doesn't render JSON anymore to dissect.